substrate-examples/offchain-worker/README.md

```diff
@@ -69,11 +69,48 @@ the warning is not generated.
 
 ## Open issues
 
-- One specific piece of code lead to a crash in MIRAI
-- For more complex scenarios timeouts arise within MIRAI. In some cases increasing the `body_analysis_timeout` parameter leads to crashes in MIRAI.
-- There are a variety of other warnings raised from code in other crates. This is confusing to the user.
-- We also tried to do the tag check directly on the `submit_price_unsigned` function as this would be closer to a real world use-case but we did not manage to get this to work correctly. We believe this is due to timeouts that MIRAI ran into (see issue above) but further investigation would be needed to verify this is indeed the problem.
-- In the debug log of MIRAI there are multiple log messages regarding a bug:
+> When reproducing the examples, make sure to run `cargo clean` after changing the [.cargo/config.toml](.cargo/config.toml).
+
+- One specific piece of code leads (see line 745 in `src/lib.rs`) to a crash in MIRAI. This happens only on the `devvirtualbox` test machine and could not be reproduced on the second system.
+![MIRAI_CRASH_CODE](mirai-crash-code.png)
+
+- For more complex scenarios timeouts arise within MIRAI. 
+  - For example with an empty `config.toml` file. See the `README.md` of `pallet_template` regarding this.
```

Comment:
you sometimes write `config.toml` and sometimes use a link. Can you do it the same way everywhere? (I would prefer the link)